SQL/0000-00-00-schema.sql

```diff
@@ -902,7 +902,8 @@ INSERT INTO `notification_types` (Type,private,Description) VALUES
     ('tarchive loader',1,'calls specific Insertion Scripts'),
     ('tarchive validation',1,'Validation of the dicoms After uploading'),
     ('mri upload runner',1,'Validation of DICOMS before uploading'),
-    ('mri upload processing class',1,'Validation and execution of DicomTar.pl and TarchiveLoader');
+    ('mri upload processing class',1,'Validation and execution of DicomTar.pl and TarchiveLoader'),
+    ('imaging non minc file insertion', 1, 'Insertion of non-minc files into the mri-table');
```

Comment:
what's "the mri-table"?

Reply:
yeah, I basically copied over the description for the "minc insertion" script and changed it to say "non-minc" instead of "minc" (see line 901 of the schema).

I could modify the description of both to say something like that instead?
"Insertion of non-minc files into the MRI tables (files/parameter_file)"

@driusan Let me know what you think. Thanks!!

Reply:
that one's more understandable.. I just don't understand what the current version means.